docs/introduction/events.md

```diff
@@ -1,26 +1,29 @@
 # Events
 
-Events fall into two categories:
-- journal: tasks in the journal
-- project: tasks in other notes
+Events are any blocks in your database that have both a time dimension and a Logseq task property associated with them (e.g. TODO, DOING, LATER, NOW, WAITING)
 
-## Journal
+Events can be in your journal or in any other page. Note that there are slight differences in setting up events in the journal vs in other pages. 
 
-Use the date of the diary as the date of the task, and when the task starts with a time, treat it as the specific time of the task.
+Also note that if you want events to show up in your project dashboard, you have have to insert those blocks in the relevant project page.
 
-Example:
-![journal tasks](../../screenshots/journal-tasks.png)
+## Journal page only
+
+Enter the task on the journal page for that date. If the task starts at a certain time, specify the time at the start of the block. You can also specify the end time, or alternatively the plugin will use your default time for events when displaying the event in your calendar.
```

Comment:
Is this the right place to introduce the idea of using the #milestone tag as well?   I'll confess that I don't quite get how a "milestone" is different than an "event", but based on this video https://www.youtube.com/watch?v=m2RFQHmA-zA this seems to be another way of creating events on the journal page.

Reply:
I will add a separate note about milestone later.